src/ImageSharp/Formats/Jpeg/Components/Decoder/JpegPixelArea.cs

```diff
@@ -69,21 +66,16 @@ public JpegPixelArea(byte[] pixels, int striede, int offset)
 
         /// <summary>
         /// Creates a new instance of the <see cref="JpegPixelArea" /> struct.
-        /// Pixel array will be taken from a pool, this instance will be the owner of it's pixel data, therefore
-        /// <see cref="ReturnPooled" /> should be called when the instance is no longer needed.
+        /// Pixel array will be handled by <see cref="Buffer2D{T}"/>, but
+        /// <see cref="ReturnPooled" /> can be called when the instance is no longer needed.
         /// </summary>
         /// <param name="width">The width.</param>
         /// <param name="height">The height.</param>
         /// <returns>A <see cref="JpegPixelArea" /> with pooled data</returns>
-        public static JpegPixelArea CreatePooled(int width, int height)
-        {
-            int size = width * height;
-            byte[] pixels = BytePool.Rent(size);
-            return new JpegPixelArea(pixels, width, 0);
-        }
+        public static JpegPixelArea CreatePooled(int width, int height) => new JpegPixelArea(Buffer2D<byte>.CreateClean(width, height), width, 0);
```

Comment:
We should remove `CreatePooled()` / `ReturnPooled()` entirely, because this is an ownership/lifecycle concern. `JpegDecoderCore` and `YCbCrImage` should own (create + dispose) `Buffer2D<byte>` instead. 
`JpegPixelArea` has to be a temporal struct to manage a (sub)area of pixels owned by `Buffer2D<byte>`.